### PR TITLE
Fixes #828, #802, #799, and #796

### LINKED ISSUE #828: Emulator: Azure sign in modal, remove Emulator menu bar from window
Emulator menu bar is appearing in the sign in window. Suppress/remove the menubar (but keep the close button):
screenshot 76 -cropped_li

### PR DESCRIPTION
**Pull request by justinwilaby**
#828 #802  #799 #796

### CODE REVIEW

File: packages/app/client/src/ui/dialogs/service/dialogService.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+describe('The DialogService', () => {
+  it('should resilve to null if no dialpogHost elemeent is set', async () => {
+    const result = await DialogService.showDialog(mockComponent);
+    expect(result).toBeNull();
+  });
+
+  it('should render the component to the host element and notify the store when "showDialog" is called', async () => {
+    const hostElement = document.createElement('div');
+    const dispatchSpy = jest.spyOn(mockStore, 'dispatch');
+    let renderedToElementEvent = undefined;
```
Comment (corinagum) [inline packages/app/client/src/ui/dialogs/service/dialogService.spec.tsx]: For other tests in this PR, [variables aren't explicitly set to undefined](https://github.com/Microsoft/BotFramework-Emulator/pull/829/files#diff-9257fdedd5971530b38c6024e4711d9aR70).

File: packages/app/client/src/ui/dialogs/service/dialogService.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  }
+};
+const mockStore = createStore(combineReducers({ resources, bot }));
+jest.mock('../../../data/store', () => ({
+  get default() {
+    return mockStore;
+  }
+}));
+describe('The DialogService', () => {
+  it('should resilve to null if no dialpogHost elemeent is set', async () => {
```
Comment (corinagum) [inline packages/app/client/src/ui/dialogs/service/dialogService.spec.tsx]: nitpick: 'should resolve to null if no dialogHost element...'

File: packages/app/client/src/ui/dialogs/tabManager/tabManager.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  });
+
+  it('should set event listeners on the window object when the component mounts', () => {
+    expect(windowEvents).toEqual([
+      'keydown',
+      'keyup'
+    ]);
+  });
+
+  describe('should set the state as expected when the', () => {
```
Comment (corinagum) [inline packages/app/client/src/ui/dialogs/tabManager/tabManager.spec.tsx]: description is unclear 😄

File: packages/app/client/src/ui/dialogs/tabManager/tabManager.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      expect(instance.state.selectedIndex).toBe(2);
+    });
+
+    it('"ArrowUp" key is pressed with the selected index not zero', () => {
+      instance.setState({ showing: true, selectedIndex: 1 });
+      instance.onKeyDown({ key: 'ArrowUp' });
+      expect(instance.state.selectedIndex).toBe(0);
+    });
+
+    it('"DownArrow key is pressed with the selectedIndex not zero', () => {
```
Comment (corinagum) [inline packages/app/client/src/ui/dialogs/tabManager/tabManager.spec.tsx]: missing closing "

File: packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.tsx [hunk 5]
```diff
@@ -86,33 +101,63 @@ export class EndpointEditor extends Component<EndpointEditorProps, EndpointEdito
   }
 
   public render(): JSX.Element {
-    const { endpointService, appIdError, appPasswordError, endpointError, nameError, isDirty } = this.state;
+    const {
+      endpointService,
```
Comment (corinagum) [inline packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.tsx]: alphabetical order?

File: packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.scss [hunk 6]
```diff
@@ -38,3 +38,62 @@
   font-size: 11px;
   line-height: 11px;
 }
+
+.arrow {
+  position: relative;
```
Comment (corinagum) [inline packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.scss]: Does this project have policies where scss should be in alphabetical order (or some other order)?

File: packages/app/client/src/ui/dialogs/service/dialogService.spec.tsx (hunk 1 above)
Comment (justinwilaby) [reply packages/app/client/src/ui/dialogs/service/dialogService.spec.tsx]: this is a lint warning where the `let` should be initialized to a value. Here, the value is `undefined`. its redundant for sure but I'm not going to argue with Mr Lint

File: packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.tsx (hunk 5 above)
Comment (justinwilaby) [reply packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.tsx]: ..yes or the order in which they are used.

File: packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.scss (hunk 6 above)
Comment (justinwilaby) [reply packages/app/client/src/ui/shell/explorer/endpointExplorer/endpointEditor/endpointEditor.scss]: No. I think that could be a good policy to implement but only with auto-formatting or as a build task.

**Pull request merged**